src/r4300/r4300_core.h

```diff
@@ -31,4 +37,30 @@ struct r4300_core
 
 void init_r4300(struct r4300_core* r4300);
 
+int64_t* r4300_regs(void);
+int64_t* r4300_mult_hi(void);
+int64_t* r4300_mult_lo(void);
+unsigned int* r4300_llbit(void);
+uint32_t* r4300_pc(void);
+
+uint32_t* r4300_last_addr(void);
+unsigned int* r4300_next_interrupt(void);
+
+unsigned int get_r4300_emumode(void);
```

Comment:
The rule I used to name my functions here is a little bit more complicated than that :
- for "normal" functions I put the action verb first (-> get_r4300_emumode, write_ai_regs, invalidate_cached_code ...)
- for "thin" RW wrappers that just return a pointer to a variable I did not put any verb that's why you get the r4300 first (r4300_regs, r4300_mult_hi, ...). Since these are used to read AND write the pointed variables (the returned pointer is not const) I felt like putting a get_ verb prefix would be somewhat misleading (at least wrt the get/set pattern). And I don't really want to write both getter/setter functions for these : a single function which returns a pointer is good enough here.

That's the somewhat rationale behind the function names. I hope it makes sense.